cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature

```diff
@@ -6,52 +6,117 @@ Feature: SecureStorage_Manage
+        And 'Firebolt' platform responds to '1st party app' for 'null for updating a secure data value for an app'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        Then 'Firebolt' platform responds with '<Validation_Key2>'
+        When 1st party app invokes the 'Firebolt' API to '<Remove_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for '<Remove_API_Key>'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        Then 'Firebolt' platform responds with '<Validation_Key2>'
 
         Examples:
-            | Scenario                        | Set_API_Key                                            | API_Key                                                      |
-            | removeForApp with device scope  | remove secure data value for an app with scope device  | get stored value for authTestTokenDevice with scope device   |
-            | removeForApp with account scope | remove secure data value for an app with scope account | get stored value for authTestTokenAccount with scope account |
+            | Scenario                        | Remove_API_Key                                          | Set_API_Key1                                         | Set_API_Key2                                         | Get_API_Key1                                                             | Get_API_Key2                                                             | Validation_Key2                                                       | Validation_Key1                                                       | Clear_API_Key                                       |
+            | removeForApp with device scope  | remove secure data value for an app with scope device   | set secure data value for an app with scope device1  | set secure data value for an app with scope device2  | get stored value for an app for authTestTokenDevice1 with scope device   | get stored value for an app for authTestTokenDevice2 with scope device   | expected value for authTestTokenDevice2 stored data in securestorage  | expected value for authTestTokenDevice1 stored data in securestorage  | clear stored value with scope as device for an app  |
+            | removeForApp with account scope | remove secure data value for an app with scope account1 | set secure data value for an app with scope account1 | set secure data value for an app with scope account2 | get stored value for an app for authTestTokenAccount1 with scope account | get stored value for an app for authTestTokenAccount2 with scope account | expected value for authTestTokenAccount2 stored data in securestorage | expected value for authTestTokenAccount1 stored data in securestorage | clear stored value with scope as account for an app |
 
```

Comment:
we cannot use device1/ device2/ account1/ account2 when we are not actually using device1 /account1 etc. All we are testing is on a single device with a single app .

Reply:
We have added TCs only for single device and account.
device1, device2 ... etc were used for authenticationValue1 and authenticationValue2.
Renamed the params and validation objects.